libgnucash/app-utils/gnc-optiondb.cpp

```diff
@@ -853,6 +853,16 @@ gnc_register_taxtable_option(GncOptionDB* db, const char* section,
     db->register_option(section, std::move(option));
 }
 
+void
+gnc_register_invoice_print_report_option(GncOptionDB* db, const char* section,
+                                         const char* name, const char* key,
+                                         const char* doc_string, std::string value)
+{
+    GncOption option{section, name, key, doc_string,
+                     value, GncOptionUIType::INV_REPORT};
+    db->register_option(section, std::move(option));
```

Comment:
I didn't notice this before. *It doesn't work like that.* `GncOption` is a variant, you need to construct one of the GncOptionValue classes to pass in to the GncOption constructor.

Reply:
Not sure about this, it seems to be working OK.

Reply:
My error, it does work like that for simple option types.